stock_reorder_forecast/models/product_product.py

```diff
@@ -166,33 +166,22 @@ def calc_purchase_date(self):
                 TP.product_id AS id,
                 MAX(TP.stock_period_min) AS stock_period_min,
                 COALESCE(
-                    SUM(SOL.product_uom_qty), 0) /
-                    MAX(CASE WHEN TP.turnover_period < TP.prod_age
-                        THEN TP.turnover_period
-                    WHEN TP.prod_age > TP.turnover_period THEN TP.prod_age
-                    ELSE 1 END)
-                        AS turnover_average,
-                    MAX(TP.stock_period_max) AS stock_period_max
+                    SUM(SOL.product_uom_qty), 0
```

Comment:
Something to think about: This can be wrong if the user changes the unit of sale. In order to always have correct results here, you'll have to convert those to the products standard UOM. That's a bit complicated in SQL, but possible.

Another option would be not to do this calculation here, but to `array_agg` the sale line ids you find, and do the sum with the ORM afterwards.